### PR TITLE
Add note about additional failure mode

### PR DESCRIPTION
**Pull request by srsudar**
On a Pixel 4XL I was successfully scheduling notifications but they were
never showing. Turns out this was because I was missing some of the
release build setup. The notification was firing but not showing,
because the system errored out when it couldn't find the resource.

This commit adds a note to the README about this failure mode.

As this repository hosts two packages, please ensure the PR title starts with the name of the package that it relates to using square brackets (e.g. [flutter_local_notifications]). The contribution guidelines can be found at https://github.com/MaikuB/flutter_local_notifications/blob/master/CONTRIBUTING.md. Please review this as it contains details on what to follow when submitting a PR.

### CODE REVIEW

File: flutter_local_notifications/README.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ Note that when a full-screen intent notification actually occurs (as opposed to
...
+⚠️ Ensure that you have configured the resources that should be kept so that resources like your notification icons aren't discarded by the R8 compiler by following the instructions [here](https://developer.android.com/studio/build/shrink-code#keep-resources). Without doing this, you might not see the icon you've specified in your app's notifications, or your notifications might never show, instead silently failing when they look for a removed resource. The configuration used by the example app can be found [here](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/example/android/app/src/main/res/raw/keep.xml) where it is specifying that all drawable resources should be kept, as well as the file used to play a custom notification sound (sound file is located [here](https://github.com/MaikuB/flutter_local_notifications/blob/master/flutter_local_notifications/example/android/app/src/main/res/raw/slow_spring_board.mp3)).
```
Comment (MaikuB) [inline flutter_local_notifications/README.md:172]: One thing that I've been guilty of doing before is around having long sentences. Would you be able to break this up? Perhaps a separate sentence to state the specific scenario (scheduling notifications) and/or one that starts with something like "Worst case..."

**Review by MaikuB — CHANGES_REQUESTED** [review]
Thanks for the PR and good pick up. Left a comment to see if a minor tweak could be done to break up the sentence as it feels a bit on the long side

File: flutter_local_notifications/README.md (hunk 1 above)
Comment (srsudar) [reply flutter_local_notifications/README.md]: Sure, done.

Sorry for the double commit on the typo. This was a tricky file to edit without line breaks.

Comment (MaikuB) [reply flutter_local_notifications/README.md]: No worries

**Pull request merged**